src/shim/sdk/tests/DocumentSDK.test.ts

```diff
@@ -246,35 +246,9 @@ describe("DocumentSDK", () => {
                         },
                         [doc]
                     );
-                    done();
-                })
-                .catch((e) => fail(e.message));
-        });
-
-        it("passes through content byte arrays", (done) => {
-            ShimUtils.setSDKInitialized();
-            const doc = new Uint8Array(25);
-
-            (FrameMediator.sendMessage as jasmine.Spy).and.returnValue(
-                Future.of({
-                    message: {
-                        data: new Uint8Array([98, 87]),
-                    },
-                })
-            );
-
-            DocumentSDK.decrypt("mydoc", doc)
-                .then(() => {
-                    expect(FrameMediator.sendMessage).toHaveBeenCalledWith(
-                        {
-                            type: "DOCUMENT_DECRYPT",
-                            message: {
-                                documentID: "mydoc",
-                                documentData: doc,
-                            },
-                        },
-                        [doc]
-                    );
+                    //Ensure we cloned these bytes before passing them to the frame
+                    const passedDoc = (FrameMediator.sendMessage as jasmine.Spy).calls.mostRecent().args[0].message.documentData;
+                    expect(passedDoc).not.toBe(doc);
```

Comment:
Probably also should check `expect(passedDoc).toEqual(doc)` if that wasn't already covered.

Reply:
Ya, thats asserted about up above on line 239.